### PR TITLE
Member tracking

### PR DESCRIPTION
**Pull request by nilshoerrmann**
This pull request adds better Member tracking.

### CODE REVIEW

File: extension.driver.php [hunk 1]
```diff
@@ -200,7 +200,44 @@ public function getSubscribedDelegates()
             array(
                 'page'		=> '/login/',
                 'delegate'	=> 'AuthorPostPasswordResetRequest',
-                'callback'	=> 'parsePasswordAction'
+                'callback'	=> 'parseMembersLogin'
```
Comment (nitriques) [inline extension.driver.php]: Is this an error ? I do see why it is needed.

File: lib/class.tracker.php [hunk 2]
```diff
@@ -85,9 +80,30 @@ public static function log($item_type, $item_id, $action_type, $user_id, $timest
...
+            $username = __('Member %s', array($name));
+        }
+
+        return $username;
+    }
+
     public static function truncateValue($value, $max = 2048)
     {
-        if (General::strlen($value) > $max) {
+        if (General::strlen($value) < $max) {
```
Comment (nitriques) [inline lib/class.tracker.php]: Hey boy, how many extension where I copied that bug... sorry about this...

File: extension.driver.php (hunk 1 above)
Comment (nilshoerrmann) [inline extension.driver.php]: That looks like an error indeed.

**Comment by nilshoerrmann** [comment]
@nitriques: The wrong callback has been corrected.

**Comment by nitriques** [comment]
Thanks!

**Comment by nitriques** [comment]
Merged as d44c180199

**Comment by nitriques** [comment]
@nilshoerrmann BTW your `integration` branch is out of sync.

You can do (assuming you have both remotes origin and upstream)

```
git fetch --all
git checkout master
git pull upstream master --ff-only
git push origin master
git branch -D integration
git push origin :integration
git checkout -b integration
git pull upstream integration --ff-only
git push origin integration
```

I would recommend to always create a new branch for PR. That way you do not commit into `integration` commits that might not end up in the remote version (like now).

```
git checkout integration
git pull upstream integration
git push origin integration
git checkout -b new-feature
git push origin new-feature
```

**Pull request closed without merging**